### ISSUE TITLE
Minecraft 1.19 failed to load Narrator libraries

### ISSUE THREAD

**Issue by ppoozl** [MultiMC/Launcher#4783]
### Operating System

Windows

### Description of bug

Minecraft 1.19 failed to load Narrator libraries

### Steps to reproduce

Create new 1.19 instance and launch. The Log will show Failed to load Narrator libraries.

### Suspected cause

Mojang changed the minecraft version json. Edit json file or update MultiMC's json parser should fix it.
------------------------------------------------------------------------------------------------------------
1.18.2 json(text2speech)
![image](https://user-images.githubusercontent.com/7188938/174274863-3e74f7ca-5c55-4147-a473-c6c541f11469.png)
1.19 json
![image](https://user-images.githubusercontent.com/7188938/174275036-d1e7fa9d-24d3-46bb-b295-c208918b3e45.png)

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by MeeniMc** [comment]
simply downloading the file by hand and dropping it at the right place is not sufficient, as it is not in the classpath list anyway.

**Comment by wackoamd1** [comment]
yes have the same error [https://paste.ee/p/GUFXd](url)

**Comment by AndaPlays** [comment]
Got the same error: [debug](https://paste.ee/p/ik07O)

**Comment by Bluebulu998** [comment]
Got the same error

**Comment by Sydist** [comment]
Valid, i got the same error

**Comment by OProprioBR** [comment]
> Got the same error: [debug](https://paste.ee/p/ik07O)

A resposta está aqui: "123: Native library (win32-x86-64/SAPIWrapper_x86.dll) not found in resource path" e também "223: Native library (win32-x86-64/SAPIWrapper_x64.dll) not found in resource path".

Por alguma razão o modo narração do jogo não foi iniciado e nem pode ser configurado porque faltaram duas dll, e lendo o crash report eu achei a solução nele mesmo.

Você precisa baixar duas dll "SAPIWrapper_x86.dll" e "SAPIWrapper_x64.dll" e em seguida jogar ambas na pasta system32 do windows... Pronto.

----------------------------------------------------------------

The answer is here: "123: Native library (win32-x86-64/SAPIWrapper_x86.dll) not found in resource path" and also "223: Native library (win32-x86-64/SAPIWrapper_x64.dll) not found in resource path ".

For some reason the game's narration mode didn't start and it can't even be configured because two dlls were missing, and reading the crash report I found the solution in it.

You need to download two dlls "SAPIWrapper_x86.dll" and "SAPIWrapper_x64.dll" and then throw them both in the system32 folder of windows...

**Comment by phit** [comment]
no you should not do that
it's a bug in MultiMC not processing the changed vanilla library information

**Comment by Bluebulu998** [comment]
I did that and it works

**Comment by jikuja** [comment]
Just saying: The game does not crash because of missing those dll files. Unless you need a narrator just ignore those error messages and the game will work.

Not sure which crash logs you are reading....

**Comment by Bluebulu998** [comment]
![image](https://user-images.githubusercontent.com/85625300/181934981-439eb48e-30a5-4bf8-9640-440ebb0f2106.png)
![image](https://user-images.githubusercontent.com/85625300/181935061-51bace19-719a-4fad-bd9b-1c992aa5d3bf.png)
![image](https://user-images.githubusercontent.com/85625300/181935076-9b0467fb-c53f-4218-9b90-3fa99dfdeb36.png)

**Comment by OProprioBR** [comment]
> Just saying: The game does not crash because of missing those dll files. Unless you need a narrator just ignore those error messages and the game will work.
> 
> Not sure which crash logs you are reading....

Exactly, I myself don't like to play knowing that something is wrong, there is probably someone who uses or needs to use the narrator...

**Comment by MeeniMc** [comment]
Moving random files to windows system32 is a bad idea overall. It will work but can cause random incompatibilities later that will make your system unstable and are hard to pinpoint and scrub away.

**Comment by Bluebulu998** [comment]
so, fix the bug pls

**Comment by Bluebulu998** [comment]
2022.8.12
the latest dev version 0.7.0-develop-3524
still not fixed

**Comment by Bluebulu998** [comment]
PolyMC doesn't have this bug

**Comment by jikuja** [comment]
it will be fixed at some point. Your messages will not speed up the development process or give good feedback.

If you want to upvote this issue use 👍 on the first message instead of pinging everyone with zero-content messages.

TYVM all other users who used +1 on the first message instead of pinging everyone subscribed to the thread. You rock!

**Comment by peterix** [comment]
That is enough.
